src/Jackett.Common/Indexers/BJShare.cs

```diff
@@ -131,6 +131,12 @@ private string StripSearchString(string term, bool isAnime)
             return isAnime ? term.TrimEnd(_digits) : term;
         }
 
+        private string CleanTitle(string title)
+        {
+            return Regex.Replace(title, @"((S\d{2})?(E\d{2,4})?)$", "")
+                        .Trim().TrimEnd('-').Trim();
```

Comment:
Since we know that `TrimStart()` isn't necessary, just call an empty `TrimEnd()` to avoid checking the beginning of the string again.
```suggestion
                        .Trim().TrimEnd('-').TrimEnd();
```